<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>445</width>
    <height>215</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label_guess">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>111</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Take your guess</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_remaining">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>20</y>
      <width>111</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Time remaining</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_tipp">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>140</y>
      <width>101</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Your guess is...</string>
    </property>
   </widget>
   <widget class="QProgressBar" name="timebar">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>40</y>
      <width>118</width>
      <height>23</height>
     </rect>
    </property>
    <property name="maximum">
     <number>10000</number>
    </property>
    <property name="value">
     <number>10000</number>
    </property>
    <property name="textVisible">
     <bool>false</bool>
    </property>
    <property name="invertedAppearance">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="checkBtn">
    <property name="geometry">
     <rect>
      <x>302</x>
      <y>140</y>
      <width>111</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>Check</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="spin_guess">
    <property name="geometry">
     <rect>
      <x>157</x>
      <y>80</y>
      <width>91</width>
      <height>29</height>
     </rect>
    </property>
    <property name="keyboardTracking">
     <bool>true</bool>
    </property>
    <property name="maximum">
     <number>100</number>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>445</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
